plaidml/BUILD

```diff
@@ -44,7 +44,7 @@ plaidml_cc_library(
     name = "runtimes",
     visibility = ["//visibility:public"],
     deps = [
-        "//pmlc/rt/llvm",
+        "//pmlc/rt/llvm_cpu",
```

Comment:
This doesn't seem like a good name change to me. Why is this runtime CPU specific?

Reply:
We want namespaces to follow the directory hierarchy, and `llvm::` is already used a few times in this code.

Reply:
Please use `pmlc::rt:llvm`, to disambiguate, use `::llvm`.

Reply:
Done.  If you have the time, I'd like to better understand your rationale.  Note that this affects all of the headers in `pmlc/rt`, which now need to explicitly use `::llvm` everywhere, since they might be included in a context in which `pmlc::rt::llvm` exists.  (C++ namespace resolution being what it is, I find it useful to try to keep my sub-namespace names globally unique where possible.)

Reply:
I have a few intuitions:
* Maybe we should do away with nested namespaces, perhaps `pmlc` is sufficient (`mlir` fine for MLIR so far).
* Maybe we could call the namespace `llvm_rt`? It doesn't seem CPU-specific to me, but maybe I'm misunderstanding the purpose of this library.


Separately, it'd be really nice if we could somehow retain something in the spirit of the old `rt` library, which contained only runtime functions that were linked into the resulting binary. It's possible that we might need to dynamically link in the runtime functions, and so having a library dedicated to these functions only seems good. I feel like we're starting to mix in the concept of the runtime itself from the original `rt` of just functions that are linked at runtime. I can see how it gets blurry with things like vulkan.

Reply:
Interesting idea.  So basically: the output of target compilation is essentially native code, and we can either JIT it or write it to a platform-specific image format and dynamically load it, and then it'll just do the right thing?

Assuming I've got it right -- I think we could get there.  LMK if I've got the idea; in the meantime, I'll mark this as a draft again in case I want to change the approach a bit before landing it.

Reply:
It'd be nice is my way of saying, let's do this in post, no need to draft it, let's get something landed :)

Reply:
The thing is, if we're going in that direction, I want to think more about this entire chunk of code before landing it; there're a few things that I think I might want to undo.  :-)